sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 - Added `Metadata`, `AccessType`, `HasImmutabilityPolicy`, `HasLegalHold`, `LeaseDuration`, `LeaseState` and `LeaseStatus` to `FileSystemItem`.
 - Added new type `LeaseDurationType` to indicate if a lease duration is fixed or infinite.
 - Added `RequestId` in each return type for REST API calls, except for concurrent APIs.
+- Added `UpdateAccessControlRecursiveSinglePage` to update the access control recursively for a datalake directory.
+- Added `RemoveAccessControlRecursiveSinglePage` to remove the access control recursively for a datalake directory.
```

Comment:
It used to be `SetAccessControl...` right? Why rename? to align with some other SDK or suggested by somebody?

Reply:
Yes, it was Ted's comment and it aligns with .NET.